core/domain/stats_services.py

```diff
@@ -122,6 +169,14 @@ def record_answers(
         submitted_answer_list):
     """Optimally record a group of answers using an already loaded exploration..
     The submitted_answer_list is a list of SubmittedAnswer domain objects.
+
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: int. The version of the exploration.
```

Comment:
Please check to see whether this should be an int or str. (Don't automatically assume the latter -- it needs to be verified.)